package.json

```diff
@@ -28,19 +28,19 @@
   },
   "homepage": "https://github.com/usds/us-forms-system#readme",
   "devDependencies": {
```

Comment:
An `npm update` slipped in here, I can back it out if needed. The only related change here is that `raven-js` was removed. If the app wanted it they would name it directly as a `devdependency`.